dimod/binary/binary_quadratic_model.py

```diff
@@ -2145,7 +2356,12 @@ def to_serializable(self,
 
     def update(self, other: 'BinaryQuadraticModel'):
         """Add the variables, interactions, offset and biases from another
-        binary quadratic model."""
+        binary quadratic model.
+
+        Args:
+            other:
```

Comment:
Given the type annotation, this one feels especially redundant.

Reply:
This is for consistency: otherwise it looks like we missed descriptions for this method's params

Reply:
I am not sure anyone is checking them for consistency in that way, just whether they have useful information.

Reply:
Not checking for consistency, but if a users has been through a few methods and then hits one with the parameters undescribed it looks like neglect. 
Another advantage is that the alternative is endless judgment calls on how self-descriptive is every parameter name, and we'll all differ on that
I try to keep them short in such cases to save the code file's real estate